.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,5 +1,5 @@
 connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
-connection.project.dir=
+connection.project.dir=../xtext-umbrella.git/releng/gradle-composite
```

Comment:
I have a special setting for github cloning in my oomph user preferences. Oomph clones to ~/git/<reponame>-<branchname>

So on my machine it would be ../xtext-umbrella-master/releng/gradle-composite

And as far as I recall the discussion about at least lib and core projects, they should be importable with buildship only, without any oomph setup